src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -713,6 +713,10 @@ internal static void ManagePortConnectionEvents(Step stepInfo, StepUIAutomation
 
         private static void OutputPortViewModel_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
         {
+            if (CurrentExecutingStep.UIAutomation == null
+                || CurrentExecutingStep.UIAutomation.Count() < 3
+                || CurrentExecutingStep.UIAutomation[2].Parameters == null) return;
```

Comment:
I forgot to ask why do we target `UIAutomation[2]` specifically? This makses this function less generic seems

Reply:
It's in the sequence defined in the JSON file. 
Gets this parameter:
```
{
		"Sequence": 3,
		"ControlType": "function",
		"Name": "RemovePortConnector",
		"Parameters": [ "NextButton", "outputPortControl", "CoordinateSystem.ByOrigin" ],
		"Action": "execute"
	}
```

Reply:
Cool, is there a way to not hard code `UIAutomation[2]`? Is it in relation with current step number? e.g. current step number is 4 so that we should look for `sequence 3`

Reply:
Good tip! I will look for the sequence. Thanks.